src/Examine.Core/BaseIndexProvider.cs

```diff
@@ -78,7 +81,11 @@ protected ValueSetValidationResult ValidateItem(ValueSet item)
         /// </summary>
         /// <param name="values"></param>
         public void IndexItems(IEnumerable<ValueSet> values)
-            => PerformIndexItems(values.Where(x => ValidateItem(x).Status != ValueSetValidationStatus.Failed), OnIndexOperationComplete);
+            => PerformIndexItems(
```

Comment:
I'm assuming this is the fix here? What is the reason it wasn't working before?

Reply:
Yea sorry, I didn't notice all the formatting, I really should stop using ctrl + k + d 🙈 
Yea it seems like we never used the return Value from `ValidateItem` as we were just using the list where Validation did not failed.
This should now Selects all using the return values, and then returns the validated ValueSets, atleast if I'm understanding Examine correctly (I just learned what examine is and what it does) 👍

Reply:
Thanks! Yes I can see the problem was due to this change in v3 https://github.com/Shazwazza/Examine/issues/276

Before the validation was just filtering the ValueSet directly but now we create a new filtered value set.